platform/keyword-registry/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -1,4 +1,5 @@
-{{!--
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This diff is very weird but I double checked and the files are fine...

Reply:
We should try to fix it if possible so the history is correct. This should be a delete + a new file rather than a file move.

Reply:
I think deleting the file and adding it back did the job. Thanks!